assets/src/components/cluster/pods/Pods.tsx

```diff
@@ -111,8 +111,8 @@ export default function AllPods() {
       if (!data?.podDelta) return prev
       const { podDelta: { delta, payload } } = data
 
-      if (delta === 'CREATE') return { ...prev, cachedPods: uniqBy([payload, ...prev.cachedPods], p => `${p.metadata.name}:${p.metadata.namespace}`) }
-      if (delta === 'DELETE') return { ...prev, cachedPods: prev.cachedPods.filter(p => !isEqual(p, payload)) }
+      if (delta === 'CREATE') return { ...prev, cachedPods: uniqBy([payload, ...prev.cachedPods ?? []], p => `${p.metadata.name}:${p.metadata.namespace}`) }
+      if (delta === 'DELETE') return { ...prev, cachedPods: prev.cachedPods?.filter(p => !isEqual(p!, payload)) }
```

Comment:
Oh thanks! I was just about to go through and fix all these TS errors, and now I see you've taken care of it. :)

Reply:
Ye, I also added `tsc` call to the `build` script.